packages/cli/src/checks/utils.ts

```diff
@@ -52,7 +50,7 @@ type AllCheck<ErrorType> = {
   validate: (
     workspace: Package,
     allWorkspaces: Map<string, Package>,
-    rootWorkspace: Package,
+    rootWorkspace: Package | undefined,
```

Comment:
The only rule that depends on this is `INCORRECT_REPOSITORY_FIELD`. Kinda a bummer to keep providing `rootWorkspace` just for it - but on the other hand it's now the only thing that is responsible for reading this information so the implementation for this check is self-contained.